src/features/posts/postSlice.js

```diff
@@ -101,6 +102,28 @@ export const fetchDeleteComment = createAsyncThunk("post/fetchDeleteComment", as
         console.log(error);
     }
 })
+
+export const fetchLikePost = createAsyncThunk("post/fetchLikePost", async ({token, postId}) => {
+    try {
+        const response = await postLikeService(token, postId);
+        return response.data;
+
+    } catch (error) {
+        console.log(error);
```

Comment:
You could console your error as console.error that way your errors would appear separately from your logs.